tests/TestCase/Date/TimezoneTest.php

```diff
@@ -27,10 +27,8 @@ class TimezoneTest extends TestCase
      */
     public function testNoopOnTimezoneChange()
     {
-        $tz = new DateTimeZone('Pacific/Honolulu');
         $date = new Date('2015-01-01');
-        $new = $date->setTimezone($tz);
```

Comment:
`Date::setTimezone()` was incorrectly returning `$this` instead of a new instance.